.travis.yml

```diff
@@ -44,13 +44,14 @@ install:
   - conda info -a
 
   # Replace dep1 dep2 ... with your dependencies
-  - conda create -c conda-forge -q -n test-environment python=$TRAVIS_PYTHON_VERSION nose==1.3.7 flake8>=3.5.0 pandas==$PANDAS_VERSION pyarrow==$PYARROW_VERSION decorator
+  - conda create -c conda-forge -q -n test-environment python=$TRAVIS_PYTHON_VERSION sphinx==2.0.1 nbsphinx=0.4.2 numpydoc=0.8.0 nose==1.3.7 flake8>=3.5.0 pandas==$PANDAS_VERSION pyarrow==$PYARROW_VERSION decorator
```

Comment:
note: we gonna make a requirement.txt for dev

Reply:
just update the README.md file

Reply:
Will do. But `requirement.txt` for dev is still needed for adding other dependencies like `nose` . Otherwise, we should keep adding dependecies in this line.

Pandas has a separate file for dev too https://github.com/pandas-dev/pandas/blob/master/requirements-dev.txt

Reply:
make sense. is there a way for conda to pick up that file as well? then we don't need that readme instruction to list requirements and instead just have a conda command.

Reply:
Seems there are two ways:

 - To have `requirements.txt` for PIP and `environment.yml` for Conda (with conversion script [generate_pip_deps_from_conda.py](https://github.com/pandas-dev/pandas/blob/022f458ee858721d366c27dd88940d69383ab960/scripts/generate_pip_deps_from_conda.py)). Seems what Pandas does.

- Looks we are able to share requirements.txt to use it in Conda too (for instance see https://www.technologyscout.net/2017/11/how-to-install-dependencies-from-a-requirements-txt-file-with-conda/).

I will make `environment.yml` for now, and take a separate look later for this one.